src/history/hs.c

```diff
@@ -1214,10 +1217,17 @@ __hs_delete_key_from_pos(
         upd->start_ts = upd->durable_ts = WT_TS_NONE;
         WT_ERR(__wt_hs_modify(hs_cbt, upd));
         upd = NULL;
+        WT_STAT_CONN_INCR(session, cache_hs_remove_key_truncate);
```

Comment:
Should we set removed to true after we have called __wt_hs_modify?

Reply:
Doesn't apply anymore.